package.json

```diff
@@ -30,7 +30,14 @@
   },
   "homepage": "http://material-ui.com/",
   "dependencies": {
-    "react-draggable2": "^0.5.1"
+    "react": ">=0.14.0-rc1",
+    "react-dom": ">=0.14.0-rc1",
```

Comment:
From my tests the current configuration works fine, all tests pass. The current package.json configurations are the only configurations I've found that completely remove dependency warnings. The docs folder, if taken out of the repository should work standalone.